src/nuketesting/_cli/main.py

```diff
@@ -145,16 +145,21 @@ def main(
     like pytest does.
 
     """
-    test_run_arguments = CLIRunArguments(
-        nuke_executable=nuke_executable,
-        test_directory=test_path,
-        config=config,
-        run_in_terminal_mode=terminal,
-        pytest_args=pytest_arg,
-        runner_name=runner_name,
-    )
-
-    _run_tests(test_run_arguments)
+    try:
+        test_run_arguments = CLIRunArguments(
+            nuke_executable=nuke_executable,
+            test_directory=test_path,
+            config=config,
+            run_in_terminal_mode=terminal,
+            pytest_args=tuple(pytest_arg),
```

Comment:
Would it be possible to add a test for this? To make sure we get the proper behavior

Reply:
Oh, I just casted a list to a tuple as it is defined in the type hints for that class. There won't be a change in behaviour, I promise :D